### ISSUE TITLE
A problem about val loss

### ISSUE THREAD

**Issue by kuazhangxiaoai** [bubbliiiing/yolox-pytorch#49]
The val loss become a high peak in training process. Do you know about this situation?
Thanks!
![image](https://user-images.githubusercontent.com/54108453/146626412-f4daeebf-40d1-4a9e-b890-fbb204a3561f.png)

**Comment by bubbliiiing** [comment]
我感觉应该是因为数据集和预训练相似性太低，然后突然崩了。问题不大一般。后面会降
